lib/ex_aws/sqs/client.ex

```diff
@@ -0,0 +1,122 @@
+defmodule ExAws.SQS.Client do
+  use Behaviour
+
+  @moduledoc """
+  The purpose of this module is to surface the ExAws.SQS API tied to a single
+  configuration chose, such that it does not need passed in with every request
+
+  Usage:
+  ```elixir
+  defmodule MyApp.SQS do
+    use ExAws.SQS.Client, otp_app: :my_otp_app
+  end
+  ```
+
+  In your config
+  ```elixir
+  config :my_otp_app, ExAws,
+    sqs: [], #SQS config goes here
+  ```
+  """
+
+  @type sqs_permission :: 
+    :all | 
+    :receive_message | 
+    :delete_message |
+    :change_message_visibility | 
+    :get_queue_attributes 
+  @type sqs_acl :: [{binary, sqs_permission}]
```

Comment:
agreed in the above code snippet it is  `{account_id, sqs_permission_atom}` 

it's clean and makes sense to have `{sqs_permission_atom, account_id}` instead